axelrod/tests/unit/test_strategy_transformers.py

```diff
@@ -121,14 +122,6 @@ def test_StrategyReBuilder_many_decorators(self):
         new_player.__dict__.update(update_dict)
         self.assertEqual(player, new_player)
 
-    def test_all_strategies(self):
```

Comment:
I believe this was added as a regression test because of a bug we'd missed and wanted to make sure it didn't come back with a new strategy. Sure it's worth getting rid of?

Reply:
This test was moved into the class that tests the Identity transformer. I did some pretty extensive rearranging of this file. We could consider having a `TestTransformer` class like `TestPlayer` that attempts to decorate every strategy as well, but I've covered a great many unusual cases -- composed transformers, transformed meta strategies, etc. -- elsewhere in this file.

Reply:
Great. Thanks for clarifying :+1: